src/mame/philips/mcd212.h

```diff
@@ -235,6 +236,10 @@ class mcd212_device : public device_t,
 	emu_timer *m_ica_timer = nullptr;
 	emu_timer *m_dca_timer = nullptr;
 
+	// Cursor State
+	uint16_t m_blink_time; // Counter that tracks how long since the last m_blink_active last changed.
+	bool m_blink_active = false;
```

Comment:
You haven’t registered these new data members for saved states in `device_start`.  Please check that all data members that hold live state are registered for saved states.